### DISCUSSION TITLE
大佬们，使用nuclei的时候出现这个错误，该如何解决呢

### DISCUSSION THREAD

**Discussion by xiaoxie2020** [projectdiscovery/nuclei#5160] [Q&A]
<img width="682" alt="image" src="https://github.com/projectdiscovery/nuclei/assets/74695420/4022c15f-31af-42ac-a861-1cf3cd29c5e6">

**Comment by ehsandeep** [comment]
@xiaoxie2020 you need to update you nuclei to latest version - https://github.com/projectdiscovery/nuclei/releases/tag/v3.2.6

**Reply by xiaoxie2020** [reply]
明白了明白了，谢谢大佬，非常感谢
